go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/samuel/go-zookeeper v0.0.0-20190923202752-2cc03de413da
-	github.com/segmentio/kafka-go v0.4.35
+	github.com/segmentio/kafka-go v0.4.43-0.20230913165112-9ecb9d2f7da5
```

Comment:
Need to release a new version of kafka-go with the added ACLs APIs and bump the version here